python/cudf/cudf/core/column_accessor.py

```diff
@@ -341,6 +341,58 @@ def _pad_key(self, key: Any, pad_value="") -> Any:
             key = (key,)
         return key + (pad_value,) * (self.nlevels - len(key))
 
+    def replace_level_values(
+        self, mapping: dict or callable, level: int or None
+    ):
+        """
+        Returns a new ColumnAccessor with values in the keys replaced according
+        to the given mapping and level.
```

Comment:
I think we could expand this docstring a bit. In particular, we should explain the `mapping` parameter in more detail: i.e., what the behaviour is when `mapping` is a dictionary v/s a callable.